compiler/include/llvmDebug.h

```diff
@@ -33,20 +33,14 @@
 #include "llvm/Support/Dwarf.h"
 #endif
 
-#if HAVE_LLVM_VER >= 35
 #include "llvm/IR/DebugInfo.h"
 #include "llvm/IR/DIBuilder.h"
-#else
-#include "llvm/DebugInfo.h"
-#include "llvm/DIBuilder.h"
-#endif
 
 #include <vector>
 
 //#define DW_LANG_chapel (llvm::dwarf::DW_LANG_lo_user+37)
 #define DW_LANG_chapel llvm::dwarf::DW_LANG_lo_user
 
-#if HAVE_LLVM_VER >= 37
 // This should be renamed to DI versions
 #define LLVM_DITYPE llvm::DIType*
```

Comment:
I recommend we remove these #defines and static inline functions and replace their uses manually. They are a source of clutter / confusion. The only reason they were there was to support different LLVM versions.